pkg/syncer/syncer.go

```diff
@@ -299,6 +328,46 @@ func (c *Controller) handleErr(err error, i interface{}) {
 	klog.Errorf("Dropping key %q after failed retries: %v", i, err)
 }
 
+const nsDelimiter = "--"
+const nsTemplate = "kcp" + nsDelimiter + "%s" + nsDelimiter + "%s"
+
+type nameTransformer struct {
+	clusterName string
+	namespace   string
+}
```

Comment:
where is the use of this struct? Just have the two funcs directly. There is no added value.